<launch>

<arg name="dict_path"/>
<arg name="output_path"/>
<arg name="critique_topic"/>
<arg name="verbose" default="false"/>
<arg name="num_trials"/>

<node pkg="infitu" type="DictatedEvaluator.py" name="dictation" output="screen">
    <param name="interface/topic" value="$(arg critique_topic)"/>
    <param name="interface/verbose" value="$(arg verbose)"/>
    <param name="in_path" value="$(arg dict_path)"/>
    <param name="trials_per_input" value="$(arg num_trials)"/>
    <param name="output_path" value="$(arg output_path)"/>
    <rosparam>
        pad_fidelity: true
        fidelity: 0
    </rosparam>
</node>

</launch>